simtools/corsika/corsika_output.py

```diff
@@ -0,0 +1,1301 @@
+import functools
+import logging
+import operator
+import time
+from pathlib import Path
+
+import boost_histogram as bh
+import numpy as np
+from astropy import units as u
+from astropy.units import cds
+from corsikaio.subblocks import event_header, get_units_from_fields, run_header
+from eventio import IACTFile
+
+from simtools import io_handler
+from simtools.util.general import (
+    collect_data_from_yaml_or_dict,
+    convert_2D_to_radial_distr,
+    rotate,
+    save_dict_to_file,
+)
+
+
+class HistogramNotCreated(Exception):
+    """Exception for histogram not created."""
+
+
+class CorsikaOutput:
+    """CorsikaOutput extracts the Cherenkov photons information from a CORSIKA output file
+    (IACT file) using pyeventio.
+
+    Parameters
+    ----------
+    input_file: str or Path
+        Input file (IACT file) provided by the CORSIKA simulation.
```

Comment:
Note that in it is 'input file' while in line 28 it is 'output file'. Suggest to remove 'output' in line 28 and talk about 'CORSIKA IACT file' only.